microprofile/testing/mock-beans/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2024 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.microprofile.testing</groupId>
+        <artifactId>helidon-microprofile-testing-project</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>helidon-microprofile-testing-mock-beans</artifactId>
+    <name>Helidon Microprofile Testing Mock Beans</name>
+
+    <description>
+        Integration with Mock Beans to support tests with CDI injection
+    </description>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.helidon.microprofile.cdi</groupId>
```

Comment:
This should depend on CDI API only (not on Helidon MP, unless you actually use it).
Also this must be a `provided` dependency, as the application must already have it (otherwise there would be no beans to mock)